### PR TITLE
[Merged by Bors] - fix: cleanup new yaml file

### PR DESCRIPTION
**Pull request by eric-wieser**
Previously some Pr numbers would start with `mathlib4#` and others would not.

This also adds the mathlib4 filename, which will allow adding more links to the "out of sync" status page.

---

Again, not tested as this is awkward to run on a new machine.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: scripts/make_port_status.py [hunk 1]
```diff
@@ -113,9 +113,9 @@ def get_mathlib4_module_commit_info(contents):
         capture_output=True)
     pr_matches = re.search(r'(#[0-9]+)\)$', log.stdout.decode().splitlines()[-1])
     if pr_matches:
-        mathlib4_pr = 'mathlib4' + pr_matches.groups()[0]
+        mathlib4_pr = int(pr_matches.groups()[0])
```
Comment (jcommelin) [inline scripts/make_port_status.py:116]: The match starts with `#`. So this will error.

File: scripts/make_port_status.py [hunk 2]
```diff
@@ -113,9 +113,9 @@ def get_mathlib4_module_commit_info(contents):
         capture_output=True)
     pr_matches = re.search(r'(#[0-9]+)\)$', log.stdout.decode().splitlines()[-1])
```
Comment (eric-wieser) [inline scripts/make_port_status.py:114]:
```suggestion
    pr_matches = re.search(r'#([0-9]+)\)$', log.stdout.decode().splitlines()[-1])
```

File: scripts/make_port_status.py (hunk 1 above)
Comment (eric-wieser) [reply scripts/make_port_status.py:116]: Nice catch, fixed

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**